src/Web.hs

```diff
@@ -0,0 +1,45 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RankNTypes #-}
+module Web
+    ( webMain
+    , inMemoryApplication
+    )
+where
+
+import Control.Monad.RWS hiding (get)
+import Network.Wai.Middleware.Static
+import Network.Wai
+import Web.Spock.Safe
+
+import Kis
+import Kis.SqlBackend
+
+__ASSET_DIR__ :: FilePath
+__ASSET_DIR__ = "frontend"
+
+webMain :: IO ()
+webMain = runSpock 8080 inMemoryWeb
+
+web :: MonadIO m => (forall a. KisClient m a -> IO a) -> IO Middleware
+web runKisClient =
+    spockT runKisClient webInterface
+
+inMemoryWeb :: IO Middleware
+inMemoryWeb = do
+    backend <- inMemoryBackend
+    web (runClient backend)
+
```

Comment:
Ah klar, verstehe. Meine Vorstellung ist, dass wir eine Main app haben, die einmal die entsprechende Connection (oder den Pool) aufmacht und dann die einzelnen Module in eigenen Threads startet, und auch etwa den Notifications thread. Dann würden wir withKis wahrscheinlich den Connection string oder so übergeben und die Funktion würde eine Connection aufbauen, oder was meinst du?